internal/util/logging.go

```diff
@@ -24,70 +26,66 @@ const (
 	ECSLogger = "ecs"
 	DevLogger = "dev"
 
-	defaultLoggerType = ECSLogger
+	DefaultLoggerType  = ECSLogger
+	DefaultLoggerLevel = zapcore.InfoLevel
 )
 
-var logLevel = zap.LevelFlag("log-level", zap.InfoLevel, "log level (default \"info\")")
-var logType = flag.String("log-type", defaultLoggerType, "log type (ecs, dev)")
-
-var logger *zap.Logger
-var loggerMutex sync.Mutex
+var warnInvalidTypeOnce sync.Once
```

Comment:
Probably this variable could be removed. I don't see any usage of it, at least in this file (internal/util/logging.go)

Reply:
Good catch, I have added `staticcheck` to `mage check` so I don't miss these ones again :slightly_smiling_face:

Reply:
That's perfect! :+1: